projects/managed-hubs/sales.md

```diff
@@ -78,7 +78,9 @@ Set up the service
 : Once we've received an email confirmation of a particular service offering, it is time to set up the administrative and technical infrastructure for the service.
 
   - **The CS&S team will set up the admin paperwork**. They will contact the community we're working with to set up the proper billing channels.
-  - [**Create a {guilabel}`New Hub` issue**](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+hub&template=2_new-hub.yml&title=%5BNew+Hub%5D+%7B%7B+HUB+NAME+%7D%7D). This will alert the engineering team that it is time to create a new hub for this community!
+  - [**Create a {guilabel}`New Hub - Request Information` issue**](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+hub&template=3_new_hub-request-info.yaml&title=%5BRequest+Information%5D+New+Hub%3A+%7B%7B+HUB+NAME+%7D%7D). Follow the instructions in this issue template and provide a high-level description of this hub, as well as ping the community representative(s) in the appropriate section. This issue is meant to help you guide the community representative(s) and answer their questions in order to empower them to open a [**{guilabel}`New Hub - Provide Information` issue**](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+hub&template=2_new-hub-provide-info.yml&title=%Provide+Information%5D+New+Hub%3A+%7B%7B+HUB+NAME+%7D%7D).
+  Feel free to ping the engineering team whenever their input is needed.
+  - Make sure the step above concluded with a [**new {guilabel}`New Hub - Provide Information` issue**](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+hub&template=2_new-hub-provide-info.yml&title=%Provide+Information%5D+New+Hub%3A+%7B%7B+HUB+NAME+%7D%7D) and this issue is appropriately added to the Team Backlog. This will alert the engineering team that it is time to create a new hub for this community! An engineer will then be assigned to a `New Hub - Provide Information` issue in order to deploy this hub and not a `New Hub - Request Information` issue.
```

Comment:
A quick question: could we call this issue template `Request a new hub` instead of `Provide information`? The goal of that issue is to create a hub right? And we provide information as part of that process. I think this could make it easier to distinguish from the "Request information" issue.

Reply:
I believe this is a very good idea!

Reply:
@choldgraf, I've renamed it and also opened https://github.com/2i2c-org/infrastructure/pull/1797